_includes/charts/calico/templates/calico-node.yaml

```diff
@@ -346,7 +351,7 @@ spec:
               command:
               - /bin/calico-node
               - -felix-live
-{{- if and (eq .Values.network "calico") (not .Values.flannel_migration) }}
+{{- if and (eq .Values.network "calico") (not .Values.flannel_migration) (not .Values.eks) }}
```

Comment:
Should this be `(.Values.vxlan)` to be more accurate? Same with the other places we're using this - seems like it's more appropriate to toggle based on whether we're using vxlan rather than if we're on Amazon EKS

Reply:
I wasn't sure about using `vxlan==true` in general. I saw that calico-vxlan.yaml is being used only with EKS/CNI so went ahead and introduced `eks` to keep the impact to EKS/CNI use-cases.

Based on your comments, it appears `network==calico and vxlan==true` will suffice. I will propose a new change removing `eks==true` and using only `network==calico and vxlan==true` consistenly.

Thanks